purchase_order_secondary_unit/models/purchase_order.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.tools.float_utils import float_compare, float_round
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    secondary_uom_qty = fields.Float(
+        string='Secondary Qty',
+        digits=dp.get_precision('Product Unit of Measure'),
+    )
+    secondary_uom_id = fields.Many2one(
+        comodel_name='product.secondary.unit',
+        string='Secondary uom',
+        ondelete='restrict',
+    )
+
+    @api.onchange('secondary_uom_id', 'secondary_uom_qty')
+    def onchange_secondary_uom(self):
```

Comment:
@sergio-teruel Prefix your onchanges by '_'

Reply:
I have changed the name to conventions, so I merge.